jac-uk/apply
# #1940 Welsh Language Amendments

## What's included?
Closes [admin #1940](https://github.com/jac-uk/admin/issues/1940).

Note: here is the list of different translations of `Yes` and `No`.
[welsh-yes-no.xlsx](https://github.com/jac-uk/apply/files/11121359/welsh-yes-no.xlsx)


## Who should test?
✅ Product owner
✅ Developers
✅ UTG

## How to test?
Apply for a welsh vacancy and check if the amended Welsh translations show properly. 

## Risk - how likely is this to impact other areas?
🟢 No risk - this is a self-contained piece of work

## Additional context
Include screen grabs, notes etc.

---
PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_
